### PR TITLE
Add champion

### PR DESCRIPTION
**Pull request by jessealama**
I'll be working on this at Igalia for a while.

### CODE REVIEW

**Comment by jessealama** [comment]
@ptomato Is this the right way to become a champion?

**Comment by jessealama** [comment]
@sarahghp It looks like you're not a TC39 delegate anymore? Shall we remove your name from the list of decimal champions?

**Comment by ljharb** [comment]
@jessealama this seems fine. Yes, I’d remove Sarah, and as soon as an existing champion confirms, i can merge this.

**Comment by ptomato** [comment]
Confirmed.

**Comment by jessealama** [comment]
ping

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The TC39 Decimal proposal is an investigation into adding a built-in data type i
 
 This whole proposal is basically a big open question, and we'd welcome your participation in discussing the design space in the issues linked above. **We are seeking input for your needs around JavaScript decimal in [this survey](https://forms.gle/A2YaTr3Tn1o3D7hdA).**
 
-**Champions**: Sarah Groff Hennigh-Palermo (Igalia), Philip Chimento (Igalia), Andrew Paprocki (Bloomberg)
+**Champions**: Sarah Groff Hennigh-Palermo (Igalia), Philip Chimento (Igalia), Andrew Paprocki (Bloomberg), Jesse Alama (Igalia)
```
Comment (ljharb) [inline README.md:7]:
```suggestion
**Champions**: Philip Chimento (Igalia), Andrew Paprocki (Bloomberg), Jesse Alama (Igalia)
```

**Review by ljharb — APPROVED** [review]

**Pull request merged**